_posts/en/newsletters/2025-02-14-newsletter.md

```diff
@@ -0,0 +1,273 @@
+  a2link="https://bitcoincore.reviews/31363#l-116"
+
+  q3="What does it mean for a `TxGraph` to be oversized? Is that the
+  same as the mempool being full?"
+  a3="A `TxGraph` is oversized when at least one of its `Cluster`s
+  exceeds the `MAX_CLUSTER_COUNT_LIMIT`. This is not the same as the
+  mempool being full, because a `TxGraph` can have more than one
+  `Cluster`."
+  a3link="https://bitcoincore.reviews/31363#l-147"
+
+  q4="If a `TxGraph` is oversized, which functions can still be called,
+  and which ones can’t?"
+  a4="Operations that could require actually materializing an oversized
+  cluster, as well as functions that require O(n<sup>2</sup>) work or more, are
+  not allowed for an oversized `Cluster`. This includes operations such
+  as computing the ancestors/descendants of a transaction. Mutation
+  operations (`AddTransaction()`, `RemoveTransaction()`,
+  `AddDependency()`, and `SetTransactionFee()`), and operations such as
+  `Trim()` (roughly `O(n log n)`) are still allowed."
+  a4link="https://bitcoincore.reviews/31363#l-162"
+%}
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [LND v0.18.5-beta][] is a bug fix release of this popular LN node
+  implementation.  Its bug fixes are described as "important"
+  and "critical" in its release notes.
+
+- [Bitcoin Inquisition 28.1][] is a minor release of this [signet][topic
+  signet] full node designed for experimenting with proposed soft forks
+  and other major protocol changes.  It includes the bug fixes included
+  in Bitcoin Core 28.1 plus support for [ephemeral dust][topic ephemeral
+  anchors].
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #25832][] adds five new tracepoints and documentation for
+  monitoring peer connection events such as connection lifetime, reconnection
+  frequency by IP and netgroup, peer discouragement, eviction, misbehaviour, and
+  more. Bitcoin Core users that have Extended Berkeley Packet Filter (eBPF)
+  tracing enabled can hook into the tracepoints using the sample scripts
+  provided, or write their own tracing scripts (see Newsletters [#160][news160
+  ebpf] and [#244][news244 ebpf]).
+
+- [Eclair #2989][] adds support for [batched][topic payment batching]
+  [splices][topic splicing] in the router, allowing the tracking of multiple
```

Comment:
Linked again one line later and this link cannot even be distinguised from 'batched' link.

```suggestion
  splices in the router, allowing the tracking of multiple
```